application/backend/src/main/java/io/typerefinery/websight/models/components/forms/Form.java

```diff
@@ -60,5 +65,8 @@ protected void init() {
       this.id = DEFAULT_ID;
       this.module = DEFAULT_MODULE;
       super.init();
+      if(grid != null) {
+        grid.addClasses("card p-4 mb-3");
```

Comment:
why is this?

Reply:
The default classes for the Form container.

Can I make this in variable add assign them ?

Reply:
I have created DEFAULT_FORM_CLASSES and assigned all the default classes for the form.